package.json

```diff
@@ -30,7 +31,7 @@
     "@matrix-org/olm": "https://gitlab.matrix.org/api/v4/projects/27/packages/npm/@matrix-org/olm/-/@matrix-org/olm-3.2.3.tgz",
     "@microsoft/microsoft-graph-client": "3.0.1",
     "@mui/material": "5.12.1",
-    "@react-native-async-storage/async-storage": "1.23.1",
+    "@react-native-async-storage/async-storage": "2.1.2",
```

Comment:
Why are we updating this too, now? It's a major version upgrade, any API changes?

Reply:
Updated.

Reply:
AFAIS it depends on an even older version, but with a caret, so we are fine not updating: https://github.com/amplitude/Amplitude-TypeScript/blob/239368a7dc41c20ff6d4fd49c9d80d1fabc43922/packages/analytics-react-native/package.json#L65

Please drop the update.